app/models/term.rb

```diff
@@ -11,7 +11,13 @@
 #  updated_at :datetime         not null
 #
 class Term < ApplicationRecord
-  
-  scope :current, -> { where(':date BETWEEN term_start AND term_end', date: Date.today)}
+
+  validates_presence_of :code, :name, :term_start, :term_end
+
```

Comment:
Please consider renaming _term_start_ and _term_end_ to _term_start_date_ and _term_end_date_ respectively. Makes it more clear to anyone reading this code.

Reply:
Maybe it makes more sense to call these fields "classes_begin" and "classes_end" like they are called in the Academic Calendar. I'll talk to Sarah and Sheri.

Reply:
Yes, even these names are good. I would still suggest that you consider "date" word in the end. Thanks for following up with customers.

Reply:
I renamed fields in the terms table: term_start to classes_begin_date; term_end to classes_begin_date.